src/config.ts

```diff
@@ -1,9 +1,14 @@
-export const isProd = window.location.hostname !== 'localhost';
 
-export const API_BASE_URL = isProd
-  ? 'https://taro-chat-v1.herokuapp.com'
-  : 'http://localhost:4000';
+export const getBackendUrl = () => {
+  const isProd = window.location.hostname !== 'localhost';
```

Comment:
we should probably also do a check that `window` is not `undefined` here just to be safe, and just fallback to assuming `isProd = true`